docs/java-api/docs/index_.asciidoc

```diff
@@ -99,11 +99,13 @@ Note that you can also add arrays with `startArray(String)` and
 other XContentBuilder objects.
 
 If you need to see the generated JSON content, you can use the
-`string()` method.
+`Strings.toString()` method.
 
 [source,java]
 --------------------------------------------------
-String json = builder.string();
+import org.elasticsearch.common.Strings;
+
+String json = Strings.toString(builder);
```

Comment:
it is odd that this is not a properly tested snippet, if it was we would have caught this problem.  but I guess that the java API docs is not that all made with proper code snippets?

Reply:
Yeah, I leave that for a future PR though :)